mlflow/utils/_capture_modules.py

```diff
@@ -95,6 +95,27 @@ def __exit__(self, *_, **__):
         importlib.import_module = self.original_import_module
 
 
+class _CaptureImportedModulesForHF(_CaptureImportedModules):
+    """
+    A context manager to capture imported modules by temporarily applying a patch to
+    `builtins.__import__` and `importlib.import_module`.
+    Used for 'transformers' flavor only.
+    """
+
+    def __init__(self, disable_tf=False):
```

Comment:
```suggestion
    def __init__(self, module_to_throw):
```

Instead of a flag, can we use the module to throw?